modules/microshift-updating-rpms-y.adoc

```diff
@@ -0,0 +1,67 @@
+//Module included in the following assemblies:
+//
+//*  microshift_updating/microshift-update-rpms.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-updating-rpms_{context}"]
+= Applying minor-version updates with RPMs
+
+Updating a {product-title} minor version on a non-OSTree system is a simple matter of downloading then updating the RPMs. For example, use the following procedure to upgrade from 4.14 to 4.15.
+
+[IMPORTANT]
+====
+You can only update {product-title} from one version to the next in sequence. Jumping minor versions is not supported. For example, must update 4.14 to 4.15.
+====
+
+.Prerequisites
+* The system requirements for installing {product-title} have been met.
+* You have root user access to the host.
+* The version of {product-title} you have is compatible to upgrade to the version you are preparing to use.
+* You have verified that your host operating system is compatible with the version of {product-title} you are preparing to install.
+* You have completed a system backup.
+//* You have stopped {product-title} and any associated workloads.
+
+[NOTE]
+====
+You cannot downgrade {product-title} with this process. Downgrades are not supported.
+====
+
+.Procedure
+
+. As a root user, enable the {product-title} repositories by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-{rhel-major}-$(uname -m)-rpms
+----
+
+. Update the {product-title} RPMs by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf update microshift
+----
+
+. Reboot the host to apply updates by running the following command:
++
+[source,terminal]
+----
+# reboot
+----
+
+.Verification
+. Optional. Start a Greenboot health check to verify system health by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart greenboot-healthcheck
```

Comment:
Restarting `greenboot-healthcheck` is not necessary because the service will start and do the health assessment after the reboot

Reply:
Is there any Verification step we can give in that case?

Reply:
Checking `systemctl status greenboot-healthcheck` if it ended/exited successfully and checking its logs with `journalctl -u greenboot-healthcheck`